locksmithing.lic

```diff
@@ -0,0 +1,116 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#locksmithing
+=end
+
+custom_require.call(%w[common common-items drinfomon equipmanager])
+ 
+class Locksmithing
+  include DRC
+  include DRCI
+  def initialize
+    settings = get_settings
+    @equipment_manager = EquipmentManager.new(settings)
+    @daily_trainer ||= settings.have_training_box
+    @box = settings.picking_lockbox || 'training box'
```

Comment:
💡 Default value shouldn't be necessary since you updated `base.yaml`.

Reply:
Ahh some of these were in the base script I started with and I didn't look too close. editing now.